dp/drivers/i40e.c

```diff
@@ -81,12 +81,18 @@
 
 static void allmulticast_enable(struct ix_rte_eth_dev *dev)
 {
-	panic("allmulticast_enable not implemented");
+	rte_eth_allmulticast_enable(dev->port);
 }
 
 static void dev_infos_get(struct ix_rte_eth_dev *dev, struct ix_rte_eth_dev_info *dev_info)
 {
-	panic("dev_infos_get not implemented\n");
+	struct rte_eth_dev_info dpdk_dev_info;
+
+	rte_eth_dev_info_get(dev->port, &dpdk_dev_info);
+	dev_info->nb_rx_fgs = 128;
```

Comment:
It is related to RSS. It is hardware-specific. ixgbe has 128 flow groups. How many does the i40e has? Can we get this number using a DPDK API call?